launcher/config.go

```diff
@@ -291,15 +294,7 @@ func configurePropagators(c *Config) error {
 }
 
 func newResource(c *Config) *resource.Resource {
-	// workaround until the following change is released
-	// https://github.com/open-telemetry/opentelemetry-go/pull/1042
-	reset := false
-	if len(os.Getenv("OTEL_RESOURCE_LABELS")) == 0 {
-		reset = true
-		os.Setenv("OTEL_RESOURCE_LABELS", os.Getenv("OTEL_RESOURCE_ATTRIBUTES"))
```

Comment:
Please add a note about removing support for `OTEL_RESOURCE_LABELS` variable in favour of the OpenTelemetry supported `OTEL_RESOURCE_ATTRIBUTES`

Reply:
Done!